src/lib/route.ts

```diff
@@ -5,23 +5,23 @@ import { getRelativeLocaleUrl } from "astro:i18n";
 
 export enum Routes {
   Home = "/",
-  About = "/about",
-  Structure = "/structure",
+  About = "/explore",
+  Structure = "/explore/structure",
+  Leader = "/explore/leader",
```

Comment:
/explore is unconventional. 

keep it about , change the Label to explore

Reply:
Okay